buildSrc/build.gradle

```diff
@@ -232,6 +232,7 @@ if (project != rootProject) {
     if (isLuceneSnapshot) {
       systemProperty 'test.lucene-snapshot-revision', isLuceneSnapshot[0][1]
     }
+    maxParallelForks System.getProperty('tests.jvms', project.rootProject.ext.defaultParallel) as Integer
```

Comment:
`BuildPlugin.findDefaultParallel()` will return the string `auto` on non-unix systems, which works for `RandomizedRunner` but will blow up when we try to cast it to an `Integer` here. We need a different strategy here. Perhaps just falling back to `Runtime.availableProcessors()` if `defaultParallel` returns `auto`.

Reply:
good catch! thanks. I added a check for "auto" and capped available processors to 4. This is the same as what "auto" means for randomized testing.